src/wui/minimap.h

```diff
@@ -122,13 +123,16 @@ class MiniMap : public UI::UniqueWindow {
 
 	InteractiveBase& ibase_;
 	MiniMapLayer owner_button_impl_;
+	MiniMapLayer additional_button_impl_;
 	View view_;
 	UI::Button button_terrn;
 	UI::Button button_owner;
 	UI::Button button_flags;
 	UI::Button button_roads;
 	UI::Button button_bldns;
 	UI::Button button_ships;
+	UI::Button button_addtn;
+	UI::Button button_attck;
```

Comment:
Although it's nice if all variables are the same number of characters – please use more intuitive names ;)